File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        l = 0
        for r in range(len(nums)):
            if l != r and nums[r] != nums[l]:
                l += 1
                nums[l] = nums[r]
        
        return l+1

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int unique_index = 0;
        for (int i = 1; i < nums.length; i++) {
            if (nums[unique_index] != nums[i]) {
                nums[unique_index + 1] = nums[i];
                unique_index += 1;
            }
        }

        return unique_index+1;
    }
}